project.json

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
this file creates the concept of a "project" for nx at the source root, so we can define and run some root-level scripts